StructureDefinition/NHSDigital-MedicationRequest.StructureDefinition.xml

```diff
@@ -86,31 +86,18 @@
         <expression value="(courseOfTherapyType.coding.code != 'acute') or (courseOfTherapyType.coding.code = 'acute' and extension('https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-MedicationRepeatInformation').exists().not())" />
       </constraint>
     </element>
-    <element id="MedicationRequest.extension">
-      <path value="MedicationRequest.extension" />
-      <slicing>
-        <discriminator>
-          <type value="value" />
-          <path value="url" />
-        </discriminator>
-        <rules value="open" />
-      </slicing>
-      <min value="1" />
```

Comment:
You mentioned that Forge removed slicing info. However, you sure you want to remove min cardinality of 1?

Reply:
It was automatically recalculated within Forge, none of the individual slices on extension had a cardinality of 1 so there is no reason for the extension element to have a cardinality of 1 (unless you want to specify any of the below, but I can't see that being the case for this resource).